internal/sirius/payment_types.go

```diff
@@ -0,0 +1,30 @@
+package sirius
+
+import "errors"
+
+type PaymentType string
```

Comment:
I don't think this is used any more?

Reply:
Whoops, i've accidentally have committed this, I created it for adding a payment, although now i've thought about it, I don't think I need it, I can just set the html option values instead eg `<option value="PHONE"`